### ISSUE TITLE
Are there any similar kernel patches/extensions for Windows 7?

### ISSUE THREAD

**Issue by IveMalfunctioned** [vxiiduu/VxKex#39]

**Comment by dotexe1337** [comment]
no, i was working on one but modding kernel32.dll/kernelbase.dll directly doesnt seem to be possible on 7. the way vxkex does it (injecting into binaries at runtime) seems to be the best solution

**Comment by rastilin** [comment]
There was a kernel patch floating around years ago that enabled PAE for 32bit Windows 7 so people could use more than 4GB of RAM. They generated an additional kernel that you could choose to boot with at startup; which suggests that it is possible to modify the kernel and boot into it. That might be disproportionately difficult to do for any large change though.

**Comment by Saswat2005** [comment]
If it is difficult, I am going to ask win32 from MSFN who works on Windows Vista Extended Kernel, as I want Windows 7 Extended Kernel.
